app/controllers/images_controller.rb

```
@@ -1,5 +1,17 @@
 class ImagesController < ApplicationController
   def index
     @helloworld = "Hello World"
+    @image = Image.new
+  end
+
+  def create
+    @image = Image.create(create_params)
```

You should use `create!` here so that it will raise an exception (instead of just returning `false`) if the creation fails.